#!/usr/bin/python3

def raise_exception():
    raise TypeError("An intentional type exception has been raised.")


# Test the function
raise_exception()
